tests/validator/validator.cpp

```diff
@@ -2625,10 +2625,36 @@ TEST(Validator, refToUnitsByNameNeedsLinkUnitsToValidate)
     t2->setUnits("millisecond");
 
     validator->validateModel(model);
-    EXPECT_EQ(size_t(1), validator->errorCount());
+    EXPECT_EQ(size_t(1), validator->errorCount()); // KRM this will fail now since we *don't* need to link units anymore
```

Comment:
I guess this test will become moot if we were to go with your proposed change?

Reply:
Exactly - that's why I'm asking.